### ISSUE TITLE
Error: adb:format: Error: {"error":{"killed":false,"code":1,"signal":null,"cmd":"/opt/ubports-installer/resources/app.asar.unpacked/platform-tools/linux/adb -P 5037 shell echo ."},"stdout":"","stderr":"error: device unauthorized.\nThis adb server's $ADB_VENDOR_KEYS is not set\nTry 'adb kill-server' if that seems wrong.\nOtherwise check for a confirmation dialog on your device."}

### ISSUE THREAD

**Issue by wayneoutthere** [ubports/ubports-installer#1292]
*Automatically generated error report* 
UBports Installer Version: 0.4.18-beta 
Device: hammerhead
OS to install: Ubuntu Touch
Settings: {"channel":"ubports-touch/16.04/stable","wipe":false,"bootstrap":true,"format":false}
Package: deb
Operating System: Ubuntu Linux 18.04 bionic x64 
NodeJS version: v12.0.0 

Error log: https://paste.ubuntu.com//p/NPPKw549MR/

**Comment by wayneoutthere** [comment]
not sure what this is but my wifi slider button disappeared after doing a few scaling things in UTTT.  Then when I tried to reinstall the OS to get my wifi back... this error continued and wouldn't go away during installation.  Stuck here for now so reporting in case it's an installer issue.

**Comment by wayneoutthere** [comment]
Update for the records:

I tried everything including `adb kill-server`, `adb-reconnect`, etc, etc, and everything continually showed 'unauthorized' when i ran `adb devices`.

we tried: `system-image-cli --switch ubports-touch/16.04/stable`

we tried with sudo

both of these output a string with the device keys (I guess those were auth keys?)

finally, i decided to (from the UBports Recovery Page) try the 'wipe data' option and when I selected that it didn't do anything but went to a black nexus5 splash screen. 

Then I rebooted it and tried the installer, manually walking through the steps (it would still not detect it automatically).  However, with the manual installer, it started to work!

Yay.  I now have a working n5 again.

Note to self: Be very careful with the Ubuntu Touch Tweak Tool?  haha
